otel-agent-chart/templates/deployment-configmap.yaml

```diff
@@ -35,13 +32,37 @@ data:
           - key: newrelicOnly
             action: upsert
             value: 'true'
-
+    exporters:
+      otlp:
+        endpoint: "otel-gateway:4317"
+        tls:
+          insecure: true
+      otlp/newrelic:
+        endpoint: {{ .Values.newRelic.endpoint }}
+        headers:
+          api-key: {{ .Values.newRelic.apiKey }}
+      debug:
+        verbosity: detailed
+        sampling_initial: 5
+        sampling_thereafter: 200
     service:
       telemetry:
         logs:
           level: "debug"
       pipelines:
+        traces:
+          receivers: [otlp]
+          processors: [batch]
+          exporters: [otlp/newrelic, debug]
```

Comment:
no traces

Reply:
Should I just delete the traces pipeline? This is part of the gateway config so maybe customers might forward traces to the gateway for NR to ingest? If not, we can delete it.

Reply:
yes, just delete

Reply:
actually, you can comment it out and leave a note, (#uncomment if exporting traces to NR)